src/toast/schedule_sim_ground.py

```diff
@@ -3612,7 +3608,7 @@ def parse_patches(args, observer, sun, moon, start_timestamp, stop_timestamp):
                     continue
                 lon.append(lon[0])
                 lat.append(lat[0])
-                log.info(f"{patch.name,} corners:\n lon = {lon}\n lat= {lat}")
+                log.info(f"{(patch.name,)} corners:\n lon = {lon}\n lat= {lat}")
```

Comment:
Should probably be `{patch.name}` not `{(patch.name,)}`. Irrelevant.

Reply:
Fixed.